mcs/class/corlib/Test/System.IO/FileTest.cs

```diff
@@ -2709,8 +2709,10 @@ public void SymLinkLoop ()
 			File.Delete (path2);
 
 			try {
-				symlink (path1, path2);
-				symlink (path2, path1);
+				if (symlink (path1, path2) != 0)
```

Comment:
Could be just simple Assert.AreEqual (0, ...)

Reply:
FWIW I find the current one easier to read because the important part is the error number returned by `Marshal.GetLastWin32Error ()` as `symlink` only returns 0 or -1 so having that in the assert doesn't provide any value.

Reply:
fair enough